tests/integration/test_cli.py

```diff
@@ -0,0 +1,38 @@
+import os
+import sys
+import subprocess
+
+import pytest
+from chalice.utils import OSUtils
+
+CURRENT_DIR = os.path.dirname(os.path.abspath(__file__))
+PROJECT_DIR = os.path.join(CURRENT_DIR, 'testapp')
+ROOT_DIR = os.path.dirname(os.path.dirname(CURRENT_DIR))
+MAIN_FILE = os.path.join(ROOT_DIR, 'chalice', 'cli', '__init__.py')
+
+
+@pytest.fixture
+def local_app(tmpdir):
+    temp_dir_path = str(tmpdir)
+    OSUtils().copytree(PROJECT_DIR, temp_dir_path)
+    os.chdir(temp_dir_path)
+    sys.path.insert(0, ROOT_DIR)
```

Comment:
Why do we need the `ROOT_DIR` on our path?  This already handled as part of your dev process setup (i.e `pip install -e .`).  Also, this is mutating a global `sys.path` without putting it back afterwards, but I don't think we need this line to begin with.

Reply:
I was not making the assumption that they had done that. If I assume `chalice` is linked to this dev version that makes everything way easier, just wasn't sure that was an acceptable assumption.